twitter/twitterapp/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.db.models.base import Model
+
+# Create your models here.
+class User(models.Model): 
+    name = models.CharField(max_length=50)
+    joined_date = models.DateTimeField('Acc creation date')
```

Comment:
Try to always use proper English words to avoid any confusion.

```suggestion
    joined_date = models.DateTimeField('Account creation date')
```